index.html

```diff
@@ -388,6 +388,32 @@
          as specified in [[!RFC7231]] <a href='https://tools.ietf.org/html/rfc7231#section-4.3.2'>section 4.3.2</a>.
         </p>
       </section>
+
+      <section id="httpDELETE">
+        <h2>HTTP DELETE</h2>
+        <p>
+         The <code>DELETE</code> method is optional per [[!LDP]] <a href='https://www.w3.org/TR/ldp/#ldpr-HTTP_DELETE'>section
+         4.2.5</a> and this specification does not require Fedora servers to implement it.  When a Fedora server supports this method,
+         in addition to the requirements imposed on <a>LDPR</a>s within containers outlined in [[!LDP]]
+         <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'> section 5.2.5</a>, it must also follow the additional behavior outlined
+         below.
+        </p>
+        <section id="httpDELETEDepth">
+          <h3>Depth Header</h3>
+          <p>
+          A <code>Depth</code> header, as defined in [[!RFC2518]] <a href='https://tools.ietf.org/html/rfc2518#section-9.2'> section
+          9.2 </a>, MAY be included in <code>DELETE</code> requests.  Implementations MAY choose not to support all of the header's
+          values and MAY define their own default behavior if a Depth header is not present.
```

Comment:
I think two most likely useful values of `Depth` are `0` and `infinite`. As written there is no requirement that any particular `Depth` value be supported. However, I think it is a requirement to understand the `Depth` header and either follow it or throw an error -- if ignoring it were allowed then one would need yet more out-of-band understanding.

I think we should ref RFC4918 instead of RFC2518. The former obsoletes the latter.